src/ast-utils/behavior-finder.ts

```diff
@@ -0,0 +1,233 @@
+/**
+ * @license
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+// jshint node: true
+'use strict';
+import * as estraverse from 'estraverse';
+import * as docs from './docs';
+import * as esutil from './esutil';
+import * as jsdoc from './jsdoc';
+import * as analyzeProperties from './analyze-properties';
+import * as astValue from './ast-value';
+import * as estree from 'estree';
+import {Visitor} from './fluent-traverse';
+import {declarationPropertyHandlers, PropertyHandlers} from './declaration-property-handlers';
+import {BehaviorDescriptor, LiteralValue} from './descriptors';
+
+export function behaviorFinder() {
```

Comment:
Removed the interface, as it's only used in one place. Just letting the type inference work for now, class later.